test/cookbooks/cerner_splunk_ingredient_test/recipes/conf_helpers.rb

```diff
@@ -0,0 +1,24 @@
+my_config = {
```

Comment:
I want to test that they can be called from a recipe, where they previously could not be.